_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -27,8 +27,10 @@ class Parameters:
         max_check_lookbehind: timedelta
         # names of signals that are smoothed (7-day avg, etc)
         smoothed_signals: Set[str]
-        # how many days behind do we expect each signal to be
-        expected_lag: Dict[str, int]
+        # maximum number of days behind do we expect each signal to be
+        max_expected_lag: Dict[str, int]
+        # minimum number of days behind do we expect each signal to be
+        min_expected_lag: Dict[str, int]
```

Comment:
We also need to remove the code that handles the old `lag` style. I guess that's in `dynamic.py` since they were dynamic-specific settings.